.travis.yml

```diff
@@ -5,11 +5,8 @@ addons:
 
 language: php
 php:
-  - hhvm
   - 7.0
   - 5.6
-  - 5.5
-  - 7.0
```

Comment:
I think we should add the `php-7.1`, `php-7.2` and `php-7.3` tests in Travis CI build.

Reply:
That's a nice idea, will try it.